app/ui/src/scss/components/_masthead.scss

```diff
@@ -0,0 +1,88 @@
+@import 'syndesis-sass';
+
+// .pf-c-page {
```

Comment:
What's this commented out stuff for?  Ideally we should avoid committing commented out code, unless this is for enabling at some future point.  If so it'd be awesome if you could add a comment about that to the commented out block, thereby increasing the amount of commenting :-)

Reply:
Hey @gashcrumb, good catch. That's remnant code from a branch that I branched off from a previous masthead and vert nav branch. I'll remove it right now.